Database Scripts/V024__insert_data_item_statistic.sql

```diff
@@ -0,0 +1,54 @@
+INSERT INTO [dbo].[ItemStatistic] ([itemId], [statisticTypeId], [name], [description], [points])
+VALUES(SELECT [itemId] FROM [dbo].[Item] WHERE [name] = 'Mystic Carrot'), 
+     (SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'HEALTH'), 
+     'Health Boost', 
+     'Restores health when consumed', 
+     25),
+	 
+    (SELECT [itemId] FROM [dbo].[Item] WHERE [name] = 'Enchanted Banana'), 
+     (SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'ATTACK'), 
+     'Attack Boost', 
+     'Increases attack points when consumed', 
+     15),
+	 
+	(SELECT [itemId] FROM [dbo].[Item] WHERE [name] = 'Sorcerer''s Walnut'), 
+     (SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'DEFENSE'), 
+     'Defense Boost', 
+     'Increases defense points when consumed', 
+     20),
+	 
+	(SELECT [itemId] FROM [dbo].[Item] WHERE [name] = 'Fairy Fig'), 
+     (SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'RIZZ'), 
+     'Rizz Boost', 
+     'Increases Rizz points when consumed', 
+     25),
+	
+	(SELECT [itemId] FROM [dbo].[Item] WHERE [name] = 'Witch''s Watermelon'), 
```

Comment:
for these all stats one I think we need 3 inserts per item as the select statement that gives the StatisticType can only return 1 value but in this case returns 3 which can not be inserted as foreign key into this table.